bundle/windows-machine-config-operator.package.yaml

```diff
@@ -1,4 +1,4 @@
 channels:
-  - currentCSV: windows-machine-config-operator.v0.0.0
+  - currentCSV: windows-machine-config-operator.v2.0.0
```

Comment:
This change is being made as since the version is now being declared in a central location, the Makefile, the CSV is generated with the actual version in it, and not just the v0.0.0 placeholder that it used to be.
The package.yaml has to list an actual version present in the bundle, so it needed to be changed to the current WMCO version, the only one that will be in the bundle.